test.js

```diff
@@ -4,15 +4,52 @@ let config = {
     headless: false,
   },
 };
-
-//Test Case:1 - Only Authenticated user are able to create tasks
-
 puppeteer.launch(config.launchOptions).then(async (browser) => {
   const page = await browser.newPage();
-  await page.goto('https://dashboard.realdevsquad.com/src/task/index.html');
-  await page.type('#title', 'Testing', { delay: 1000 });
-  await page.click('#group', { clickCount: 1 });
-  await page.click('#submit', { clickCount: 1 });
+  await page.goto('https://dashboard.realdevsquad.com/src/task/index.html'); //   // http://127.0.0.1:5500/src/task/index.html
+
+  //Test Case:1 - Only Authenticated user are able to create tasks
+
+  await page.type('#title', 'Testing', { delay: 200 });
+  await page.waitForSelector('#group');
+  await page.click('#group');
+  await page.on('dialog', async (dialog) => {
+    await page.waitForTimeout(1000);
+    await dialog.accept();
+  });
+  await page.waitForSelector('#submit');
+  await page.click('#submit');
+  await page.waitForTimeout(4000);
+
+  //Test Case:2 - Change status to Assigned. Submit without filling assignee
+
+  await page.waitForSelector(
+    '.container > #taskForm > .inputBox:nth-child(7) > .editable > .edit-button',
+  );
+  await page.click(
+    '.container > #taskForm > .inputBox:nth-child(7) > .editable > .edit-button',
```

Comment:
Can we always expect the element that we are targeting to be at **nth-child(7)** ?

Let's make use of **data-testid** attribute to target elements

This might be helpful 👇 
https://www.educative.io/answers/what-is-the-data-testid-attribute-in-testing

Reply:
Used id tag instead to dynamically add the id to the element as we were mostly using id in puppeteer